frontend/src/MainPageGallery/MainPageMovieCard.css

```diff
@@ -12,22 +12,52 @@ button {
 
 .movie-card img {
     width: 180px;
+    border: none;
 }
 
 .movie-card {
-    margin: 1.3rem;
-    padding: 1.4rem 0;
-    width: 17rem;
+    margin-bottom: 3rem;
+    width: 250px;
+    height: 370px;
     font-weight: 200;
     /*background-color: #7FB5FF;*/
-    background-color: #F6F6F6;
+    background-color: #C4DDFF;
     border-radius: 8px;
     /*background: #8e9eab;  !* fallback for old browsers *!*/
     /*background: -webkit-linear-gradient(to top, #eef2f3, #8e9eab);  !* Chrome 10-25, Safari 5.1-6 *!*/
     /*background: linear-gradient(to top, #eef2f3, #8e9eab); !* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ *!*/
+}
+
+.card-inner {
+    padding: 1.2rem 0;
+    width: 100%;
+    height: 100%;
+    transition: transform 1s;
```

Comment:
1s seems a bit slow. as User i'd prefer to have it at smth like .2s .5s max. 
The whole length of the animation, the user can't interact with the element.